templates/vsc/common/declarative-agent-basic/m365agents.yml.tpl

```diff
@@ -17,6 +17,7 @@ provision:
     writeToEnvironmentFile:
       teamsAppId: TEAMS_APP_ID
 
+deploy:
```

Comment:
1> Do you need to use feature flag to control the visibility here to avoid behavior change before this feature ships?
2>Does this only apply to basic DA?

Reply:
1. Yes, the action `teamsApp/shareToOthers` is hidden behind feature flag. Just moved other actions to deploy stage.
2. No, other DA templates will be changed in next PR as well as YAML schema.

Reply:
So for #1, the change to move other actions to deploy stage is not related with the feature flag. Add @summzhan for awareness as we may need to address this change in release note if this is released before feature flag turned on.